app/components/ProfileInfoCard.tsx

```diff
@@ -0,0 +1,71 @@
+import { Avatar, Card, Separator, XStack, YStack } from "tamagui"
+
+import { ProfileData } from "backend/src/types/auth.types"
+
+import { getAbbreviatedName, getInitials } from "@/utils/nameFormatting"
+
+import { BodyText } from "./BodyText"
+import { HeaderText } from "./HeaderText"
+import { PressableIcon } from "./Icon"
+import { InfoSection } from "./InfoSection"
+
+interface ProfileInfoCardProps {
+  profile: ProfileData
+}
+
+export const ProfileInfoCard = ({ profile }: ProfileInfoCardProps) => {
+  const initials = profile ? `${getInitials(profile.first_name, profile.last_name)}` : ""
+  const displayName = profile
+    ? `${profile.first_name ?? ""} ${getAbbreviatedName(null, profile.last_name)}`
+    : ""
```

Comment:
now that profile isn't undefined, we can clean this up a bit: (also we already return strings from both helper funcs)

```suggestion
  const initials = getInitials(profile.first_name, profile.last_name)
  const displayName = getAbbreviatedName(profile.first_name, profile.last_name)
```